lib/common.sh

```diff
@@ -497,8 +497,10 @@ iterate(){
   local RUNS=0
   local COMMAND="$*"
   local TMP_RET TMP_RET_CODE
+  set +e
```

Comment:
Allowing errors here since we would otherwise immediately exit `iterate` and the whole point of the function is to retry until successful.

Reply:
Technically in library code you should never just set or unset any of the flags, without reading the exact value and then restoring that. Here we just assume we have -e, and toggle it back and forth, but we might not actually have it enabled in the first place.

That is one reason why we should maybe do it in caller side.